src/js/theme.ts

```diff
@@ -24,7 +24,7 @@ import initDropdownToggleMenu from './modules/ps_mainmenu';
 
 initEmitter();
 
-window.prestashop.themeSelectors = selectorsMap;
+window.Theme.selectors = selectorsMap;
```

Comment:
```suggestion
```

Reply:
and add `selectors` to the `export default`, it will be magically exported to `window.Theme`